projects/kit/src/components/divider/components/divider/divider.component.scss

```
@@ -0,0 +1,19 @@
+@use 'functions';
+
+:host {
+  display: block;
+}
+
+.divider {
```

теперь в пупе надо добавлять префикс к css классам

```suggestion
.pupa-divider {
```